infra/src/main/kotlin/fi/fta/geoviite/infra/switchLibrary/SwitchStructureDao.kt

```diff
@@ -23,6 +22,12 @@ import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate
 import org.springframework.stereotype.Component
 import org.springframework.transaction.annotation.Transactional
 
+private data class ElementFetchData(
```

Comment:
:meat_on_bone: 

Uuden datamallin insert/update. Samalla vaihdettu funkkarit upsertiksi. Structuren luonnollinen avain on sen nimi, tällä päivitettävä versio haettiin ennenkin. 

Samalla myös laitettu structuren osasten kirjoitus batch updateksi, joten tuosta ei tule enää lähellekään niin montaa round trippiä kun data päivitetään tai luetaan. Merkitys kuitenkin pienehkö koska näitä ei niin montaa ole että tuossa olisi koskaan kovin kauaa kestänyt.